VMEC/VMECplot.m

```diff
@@ -1459,7 +1461,7 @@ function update_plots(handles)
             z(:,:,end+1) = z(:,:,1);
             phi = 0:2*pi/(size(r,3)-1):2*pi;
             if (handles.rval > 1)
-                isotoro(r,z,phi,handles.rval);
+                isotoro(r,z,phi,handles.rval, 'STL');
```

Comment:
This default writing of stl files probably shouldnt be added to the main branch - perhaps there could be a button similar to Plot to file?